src/main/java/spireMapOverhaul/zones/Junkyard/Junkyard.java

```diff
@@ -0,0 +1,88 @@
+package spireMapOverhaul.zones.Junkyard;
+
+import com.badlogic.gdx.graphics.Color;
+import com.megacrit.cardcrawl.events.AbstractEvent;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import com.megacrit.cardcrawl.monsters.MonsterGroup;
+import com.megacrit.cardcrawl.monsters.exordium.*;
+import com.megacrit.cardcrawl.rewards.RewardItem;
+import spireMapOverhaul.SpireAnniversary6Mod;
+import spireMapOverhaul.abstracts.AbstractZone;
+import spireMapOverhaul.zoneInterfaces.EncounterModifyingZone;
+import spireMapOverhaul.zoneInterfaces.ModifiedEventRateZone;
+import spireMapOverhaul.zoneInterfaces.RewardModifyingZone;
+import spireMapOverhaul.zones.Junkyard.monsters.Junkbot;
+import spireMapOverhaul.zones.Junkyard.monsters.Peddler;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class Junkyard extends AbstractZone implements RewardModifyingZone, EncounterModifyingZone, ModifiedEventRateZone {
+    public static final String ID = "Junkyard";
+    private static final String JUNKBOTS = SpireAnniversary6Mod.makeID("Junkbots");
+    private static final String CULTIST_JUNKBOT = SpireAnniversary6Mod.makeID("Cultist_Junkbot");
+    private static final String MUGGER_JUNKBOT = SpireAnniversary6Mod.makeID("Mugger_Junkbot");
+    private static final String PEDDLER = SpireAnniversary6Mod.makeID("Peddler");
+
+    public Junkyard() {
+        super(ID, Icons.REWARD, Icons.MONSTER, Icons.EVENT);
```

Comment:
```suggestion
        super(ID, Icons.MONSTER, Icons.EVENT);
```
Reward icons hasn't existed for a long while now, you should update your branch from upstream master